fence/agents/azure_arm/Makefile.am

```diff
@@ -0,0 +1,17 @@
+MAINTAINERCLEANFILES= Makefile.in
```

Comment:
The build system is now a bit different but that is something what we can easilly fix.